centreon/www/install/php/Update-25.03.0.php

```diff
@@ -17,3 +17,51 @@
  * For more information : contact@centreon.com
  *
  */
+
+require_once __DIR__ . '/../../../bootstrap.php';
+require_once __DIR__ . '/../../class/centreonLog.class.php';
+
+$centreonLog = CentreonLog::create();
+
+$versionOfTheUpgrade = 'UPGRADE - 25.03.0: ';
+$errorMessage = '';
+
+// -------------------------------------------- CEIP Agent Information -------------------------------------------- //
+/**
+ * @param CentreonDB $pearDB
+ *
+ * @throws CentreonDbException
+ *
+ */
+$createAgentInformationTable = function (CentreonDB $pearDB) use (&$errorMessage): void {
+    $errorMessage = 'Unable to create table agent_information';
+    $pearDB->executeQuery(
+        <<<SQL
+            CREATE TABLE IF NOT EXISTS `centreon_storage`.`agent_information` (
+                `poller_id` bigint(20) unsigned NOT NULL,
+                `enabled` tinyint(1) NOT NULL DEFAULT 1,
+                `infos` JSON NOT NULL,
+            PRIMARY KEY (`poller_id`)
+            ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+        SQL
+    );
+};
```

Comment:
```suggestion
$createAgentInformationTable = function (CentreonDB $pearDBO) use (&$errorMessage): void {
    $errorMessage = 'Unable to create table agent_information';
    $pearDBO->executeQuery(
        <<<SQL
            CREATE TABLE IF NOT EXISTS `agent_information` (
                `poller_id` bigint(20) unsigned NOT NULL,
                `enabled` tinyint(1) NOT NULL DEFAULT 1,
                `infos` JSON NOT NULL,
            PRIMARY KEY (`poller_id`)
            ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
        SQL
    );
};
```

Tables are not always `centreon` and `centreon_storage`, using $PearDBO ensure you are using a connection to the realtime database not regarding its name

Reply:
Good to know, thanks
done !